app/ui/tests/.gitignore

```diff
@@ -0,0 +1 @@
+/elm-stuff/
```

Comment:
This shouldn't be required, as, rightly or wrongly, we already have that [rule at the root level](https://github.com/jameshopkins/webapptest/blob/master/.gitignore#L11)